src/components/notifications/Whapi.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+    <div class="mb-3">
+        <label for="whapi-api-url" class="form-label">{{ $t("API URL") }}</label>
+        <input id="whapi-api-url" v-model="$parent.notification.whapiApiUrl" type="text" class="form-control" required>
```

Comment:
Please add a placeholder how the url should look and/or add a helptext how it can be obtained.

Currently, you are appending `/messages/text` to this url might not be transparent to users.

Reply:
I'll add the placeholder.
I made the URL configurable because in the Whapi dashboard they give you both the URL and the Token.
As it's a generic URL (https://gate.whapi.cloud/) I actually think it should be safe to assume that it's always the same for everyone, but I prefer to be prepared in case they start giving out different URLs for different users

Reply:
Cannot reproduce a placeholder being added ^^

Reply:
I knew I missed something! Should I add a default value or a placeholder (and then use `https://gate.whapi.cloud/` when I call the API if the URL is empty) ?

Reply:
I would argue a default value would make more sense, but don't know if we have implemented this anywhere.
It might be nessesary to add this to the `<script` section (think I have seen something like this, don't know where)

Reply:
Done it on the api call side, so I can also strip the slash at the end if present